build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -53,11 +48,14 @@
       <SharedFrameworkPublishDirectory>$(IntermediateDirectory)/sharedFrameworkPublish</SharedFrameworkPublishDirectory>
     </PropertyGroup>
 
+    <GetCommitHash RepoRoot="$(RepoRoot)">
```

Comment:
I don't understand why this one is broken out, but CommitCount stays in GenerateBuildVersionInfo.